packages/main/cypress/specs/DateRangePicker.cy.tsx

```diff
@@ -0,0 +1,570 @@
+import "../../dist/Assets.js";
+import { setLanguage } from "@ui5/webcomponents-base/dist/config/Language.js";
+import DateRangePicker from "../../src/DateRangePicker.js";
+
+describe("DateRangePicker general interaction", () => {
+	afterEach(() => {
+		// eslint-disable-next-line
+		cy.wait(200);
```

Comment:
why do we need to wait after each test?

Reply:
There are similar timing related issues as in the `ui5-date-picker`. The timing issue is with realType in the input field as it doesn't always type the whole string.